src/redux/booking/reducer.js

```diff
@@ -6,6 +6,13 @@ export const bookingInstance = handleAction(
   null
 )
 
+export const error = handleAction(
+  'CREATE_BOOKING_FAILURE',
+  (state, action) => action.payload,
+  null
+)
+
```

Comment:
started w/ this in error selectors but felt weird to have CREATE_BOOKING_FAILURE there... thoughts?